### ISSUE TITLE
update footer information

### ISSUE THREAD

**Issue by arnoao1** [BIT-Studio-3/group-project-25-1-scade#31]
The information within the footer is outdated or irrelevant to the functionality of our website.
(e.g "The Disasters Service website 2024", that's no longer the website name and not the current year)
The Accessibility page contains no information and should be removed.
References seems irrelevant and could be removed.
"How To Use Our Website" contains outdated CSS styling and is irrelevant since we can put information on how to use our website within the About page.
Privacy Policy also contains outdated CSS.
